website/assets/styles/mixins-and-variables/feature-blocks.less

```
@@ -0,0 +1,128 @@
+// Mixins for feature-block layout patterns.
+// These handle LAYOUT only — typography, icons, and text alignment inside each
+// feature are page-specific. Add those overrides on the selectors after calling
+// the mixin.
+
+// Side-by-side: an image paired with a text block (headline + paragraph +
+// checklist or similar). Collapses to a single column below 992px.
+//
```

Stylelint (17.7.0)

[error] Unexpected empty comment (scss/comment-no-empty)

(scss/comment-no-empty)